spec/models/export/submission_entry_row_spec.rb

```diff
@@ -78,4 +78,25 @@
       expect(row.formatted_date('13/28/19')).to eq '13/28/19'
     end
   end
+
+  describe '#format_decimal strips non-numeric characters' do
+    let(:row) { Export::SubmissionEntryRow.new(entry) }
+    let(:entry) { double 'SubmissionEntry' }
+
+    it 'handles integers' do
+      expect(row.formatted_decimal(42)).to eql 42
+      expect(row.formatted_decimal(-1234)).to eql(-1234)
+    end
+
+    it 'handles floats' do
+      expect(row.formatted_decimal(42.42)).to eql 42.42
+      expect(row.formatted_decimal(-12.34)).to eql(-12.34)
+    end
+
+    it 'handles strings containing invalid characters' do
+      expect(row.formatted_decimal('  12.34   ')).to eql 12.34
+      expect(row.formatted_decimal('€5.12')).to eql 5.12
+      expect(row.formatted_decimal('- $10.99')).to eql(-10.99)
```

Comment:
None of these specs have a comma. Do we need to test for that?

Reply:
And do we want to spec what happens when we input a non-number? EG "N/A" or something?

Reply:
:+1: